sdk/storage_blobs/src/container/operations/list_blobs.rs

```diff
@@ -38,6 +39,8 @@ impl ListBlobsBuilder {
 
                 if let Some(next_marker) = continuation {
                     next_marker.append_to_url_query(&mut url);
+                } else if let Some(marker) = this.marker {
+                    marker.append_to_url_query(&mut url);
```

Comment:
if let Some(marker) = continuation.or(this.marker.as_ref()) {
    marker.append_to_url_query(&mut url);
}